test/e2e/set/scyllacluster/scyllacluster_evictions.go

```diff
@@ -0,0 +1,108 @@
+	o "github.com/onsi/gomega"
+	scyllaclusterfixture "github.com/scylladb/scylla-operator/test/e2e/fixture/scyllacluster"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	corev1 "k8s.io/api/core/v1"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = g.Describe("ScyllaCluster evictions", func() {
+	defer g.GinkgoRecover()
+
+	f := framework.NewFramework("scyllacluster")
+
+	g.BeforeEach(func() {
+		ctx := context.Background()
+
+		role := &rbacv1.Role{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "pod-eviction",
+				Namespace: f.Namespace(),
+			},
+			Rules: []rbacv1.PolicyRule{
+				{
+					APIGroups: []string{corev1.GroupName},
+					Resources: []string{"pods/eviction"},
+					Verbs:     []string{"create", "delete", "patch", "update", "get", "list", "watch"},
+				},
+			},
+		}
+
+		_, err := f.KubeAdminClient().RbacV1().Roles(f.Namespace()).Create(ctx, role, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		_, err = f.KubeAdminClient().RbacV1().RoleBindings(f.Namespace()).Create(ctx, &rbacv1.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      role.Name,
+				Namespace: f.Namespace(),
+			},
+			Subjects: []rbacv1.Subject{
+				{
+					APIGroup:  corev1.GroupName,
+					Kind:      rbacv1.ServiceAccountKind,
+					Namespace: f.Namespace(),
+					Name:      framework.ServiceAccountName,
+				},
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: rbacv1.GroupName,
+				Kind:     "Role",
+				Name:     role.Name,
+			},
+		}, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+	})
+
+	g.It("should allow one disruption", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimout)
+		defer cancel()
+
+		sc := scyllaclusterfixture.BasicScyllaCluster.ReadOrFail()
+		sc.Spec.Datacenter.Racks[0].Members = 2
+
+		framework.By("Creating a ScyllaCluster")
+		err := framework.SetupScyllaClusterSA(ctx, f.KubeClient().CoreV1(), f.KubeClient().RbacV1(), f.Namespace(), sc.Name)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		sc, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Create(ctx, sc, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the ScyllaCluster to deploy")
+		waitCtx1, waitCtx1Cancel := contextForRollout(ctx, sc)
+		defer waitCtx1Cancel()
+		sc, err = waitForScyllaClusterState(waitCtx1, f.ScyllaClient().ScyllaV1(), sc.Namespace, sc.Name, scyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, f.KubeClient(), sc)
+
+		framework.By("Should allow eviction of first pod")
+		e := &policyv1beta1.Eviction{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      getNodeName(0, sc),
+				Namespace: f.Namespace(),
+			},
+		}
+		err = f.KubeClient().CoreV1().Pods(f.Namespace()).Evict(ctx, e)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Should forbid eviction of second pod")
```

Comment:
same here, the ginkgo statements should be readable as a sentence - so you have:
`It("should allow one disruption")`
`By("Should forbid eviction of second pod")`
vs.
`It("should allow one disruption")`
`By("Refusing to evict a second pod")`